fuel-tx/src/receipt.rs

```diff
@@ -156,6 +156,51 @@ pub enum Receipt {
     },
 }
 
+impl core::fmt::Display for Receipt {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            Receipt::Call { id, .. } => write!(f, "Call({})", id),
+            Receipt::Return { id, .. } => write!(f, "Return({})", id),
+            Receipt::ReturnData { id, .. } => write!(f, "ReturnData({})", id),
+            Receipt::Panic { id, .. } => write!(f, "Panic({})", id),
```

Comment:
do you think we can display the panic reason in a meaningful way?

https://github.com/FuelLabs/fuel-vm/blob/2fc9c5d21d1f1ac1c4ec2e0878566c15f4f653db/fuel-tx/src/receipt.rs#L65

Reply:
Yes, since `PanicReason` is a simple `repr(u8)` enum, we can just debug-print it to get an output like:
```
Panic(id=0000000000000000000000000000000000000000000000000000000000000000, reason=OutOfGas)
```

Updated in https://github.com/FuelLabs/fuel-vm/pull/901/commits/7e04606ea5561615b445fd907f73d5e8cfc4c203

Reply:
nice, ty